contrib/dockerd-rootless-setuptool.sh

```diff
@@ -264,6 +264,19 @@ init() {
 		echo
 		exit 1
 	fi
+	if [ -n "$SYSTEMD" ] && ! systemctl --user -q is-active dbus; then
+		# dbus should be automatically enabled on installing dbus-user-session package
+		# (depended by docker-ce-rootless-extras), but dbus does not start automatically
+		# until the next login.
+		ERROR "dbus user session is not enabled. Run the following command and run this tool again."
```

Comment:
Curious; is this something that the systemd unit that we run could declare as dependency, and automatically take care of?

Reply:
Yes, updated